<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-UOPUOK8O" failures="0" tests="2" name="testScript.LoginPage_Test" time="13.317" errors="1" timestamp="2024-08-25T13:06:31 IST" skipped="0">
  <testcase classname="testScript.LoginPage_Test" name="verify_User_is_Able_To_Login_Using_Valid_Credentials" time="5.882"/>
  <system-out/>
  <testcase classname="testScript.LoginPage_Test" name="verify_User_is_Able_toLoginUsing_Invalid_UserName_and_Validpassword" time="7.435">
    <error message="C:\Users\jeffk\git\7RMart-SuperMarket\Automation_Project\src\main\resources\TestData.xlsx (The system cannot find the path specified)" type="java.io.FileNotFoundException">
      <![CDATA[java.io.FileNotFoundException: C:\Users\jeffk\git\7RMart-SuperMarket\Automation_Project\src\main\resources\TestData.xlsx (The system cannot find the path specified)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:216)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:111)
at utilities.Excel_Utilities.getStringData(Excel_Utilities.java:21)
at testScript.LoginPage_Test.verify_User_is_Able_toLoginUsing_Invalid_UserName_and_Validpassword(LoginPage_Test.java:46)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
]]>
    </error>
  </testcase> <!-- verify_User_is_Able_toLoginUsing_Invalid_UserName_and_Validpassword -->
  <system-out/>
</testsuite> <!-- testScript.LoginPage_Test -->
